base/shell/explorer/taskswnd.cpp

```diff
@@ -678,7 +975,147 @@ class CTaskSwitchWnd :
         return m_ButtonCount;
     }
 
-    INT AddTaskItemButton(IN OUT PTASK_ITEM TaskItem)
+    BOOL CollapseTaskGroup(PTASK_GROUP TaskGroup) {
+        ASSERT(m_IsGroupingEnabled);
+
+        if (TaskGroup->IsCollapsed)
+            return FALSE;
+
+        /* Get the items associated to this group and delete the index */
+        PTASK_ITEM TaskItem, LastTaskItem = NULL;
+        WCHAR windowText[255];
+        TBBUTTON tbBtn = { 0 };
+        INT iIndex = __INT_MAX__;
```

Comment:
that's the thing that breaks MSVC build. Either `INT_MAX` or `-1` as it seems to be done elsewhere in your code?